static/manifest.json

```diff
@@ -0,0 +1,34 @@
+
+{
+  "name": "Chrome Platform Status",
+  "short_name": "Chrome status",
+  "start_url": "/features?utm_source=web_app_manifest",
+  "display": "standalone",
+  "theme_color": "#366597",
+  "background_color": "#366597",
+  "icons": [{
+    "src": "/static/img/chromium-128.png",
```

Comment:
I need to switch these over from the chrome logo, just didn't have time to make the assets. Using the chrome logo makes it look like the chrome browser when it's on the homescreen :( We need something diff.